### PR TITLE
[ng] toh-pt0 rewritten as markdown

### PR DESCRIPTION
**Pull request by chalin**
Staged at https://angulardart-org-dev.firebaseapp.com/angular/tutorial/toh-pt0

### CODE REVIEW

File: src/angular/tutorial/toh-pt0.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+This will [reload the app](/angular/guide/setup#reload-the-app).
+As you save updates to the code, the `pub` tool detects changes and
+serves the new app.
+
+## Angular app basics
+
+Angular applications are made up of _components_.
+A _component_ is the combination of an HTML template and a component class that controls a portion of the screen. The starter app has a component that displays a simple string:
+
+<?code-excerpt "lib/app_component.dart" title linenums?>
```
Comment (kwalrath) [inline src/angular/tutorial/toh-pt0.md]: This didn't have line numbers in the original, but I guess this change was intentional?

File: src/angular/tutorial/toh-pt0.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+## The starter app's code
+
+The app contains the following core files:
+
+<code-tabs>
+  <?code-pane "lib/app_component.dart"?>
+  <?code-pane "test/app_test.dart"?>
+  <?code-pane "web/main.dart"?>
+  <?code-pane "web/index.html"?>
+  <?code-pane "web/styles.css" region="quickstart"?>
```
Comment (kwalrath) [inline src/angular/tutorial/toh-pt0.md]: The original says "web/styles.css (excerpt)"

File: src/angular/tutorial/toh-pt0.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+  - lib
+    - app_component.dart
+  - test
+    - app_test.dart
+  - web
+    - index.html
+    - main.dart
+    - styles.css
+  - pubspec.yaml
+</div>
```
Comment (kwalrath) [inline src/angular/tutorial/toh-pt0.md]: Not a showstopper, but this box has more white space at the bottom than the original one does.

File: src/angular/tutorial/toh-pt0.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+      The file that describes this Dart package (the app) and its dependencies.
+      For example, it specifies the **angular2** and **browser** packages as dependencies,
+      as well as the **angular2** transformer.
+
+      **Note:** The **dart_to_js_script_rewriter** transformer, if present,
+      must be **after angular2** in the **transformers** list.
+      If the order is wrong, Angular templates don't work.
+-->
+
+## What's next
```
Comment (kwalrath) [inline src/angular/tutorial/toh-pt0.md]: Not necessarily a problem, but there's less white space before this heading than there is in the original.

File: src/angular/tutorial/toh-pt0.md (hunk 3 above)
Comment (kwalrath) [inline src/angular/tutorial/toh-pt0.md]: And of course the order is different.

File: src/angular/tutorial/toh-pt0.md (hunk 1 above)
Comment (chalin) [inline src/angular/tutorial/toh-pt0.md]: Yes.

File: src/angular/tutorial/toh-pt0.md (hunk 2 above)
Comment (chalin) [inline src/angular/tutorial/toh-pt0.md]: Oh, right! Fixed.

File: src/angular/tutorial/toh-pt0.md (hunk 3 above)
Comment (chalin) [inline src/angular/tutorial/toh-pt0.md]: Right. I think that alphabetical order is (generally) better.

Comment (chalin) [inline src/angular/tutorial/toh-pt0.md]: It does, but it has been a challenge to tweak. I'll give it another try.

File: src/angular/tutorial/toh-pt0.md (hunk 4 above)
Comment (chalin) [inline src/angular/tutorial/toh-pt0.md]: I can add it back if you'd like it, but it seemed ok without that extra space.

File: src/angular/tutorial/toh-pt0.md (hunk 3 above)
Comment (chalin) [inline src/angular/tutorial/toh-pt0.md]: Fixed.

**Comment by chalin** [comment]
The issues you pointed out have been fixed (except for the extra space before the "What's next" heading). Since this has been approved, I'll merged it. Tweaks can be made later.

**Pull request merged**